src/include/hip/rpp_hip_common.hpp

```diff
@@ -1330,40 +1354,41 @@ __device__ __forceinline__ void rpp_hip_pack_float24_pln3_and_store24_pkd3(uchar
 
 __device__ __forceinline__ void rpp_hip_pack_float24_pln3_and_store24_pkd3(float *dstPtr, d_float24 *dst_f24)
 {
-    d_float24 *dstPtr_f24;
-    dstPtr_f24 = (d_float24 *)dstPtr;
+    d_float24 dstPtr_f24;
+
+    dstPtr_f24.x.x.x = dst_f24->x.x.x;
```

Comment:
also change this function to assign dstPtr_f24.x.x, dstPtr_f24.x.y, ... as float4

Reply:
Done